readme.markdown

```diff
@@ -148,8 +148,10 @@ Available `opts` options are:
 - opts.timeout = 500. Set a timeout for the test, after which it will fail.
   See test.timeoutAfter.
 - opts.objectPrintDepth = 5. Configure max depth of expected / actual object
-  printing.
-  
+  printing. If set to "-1", entire object will be printed as a well-formatted
```

Comment:
Why `-1` when JS has the value `Infinity`?

Reply:
The most useful part of the PR is probably the ability to set a global depth via the environmental variable. I don't think environmental variables have value for Infinity, do they?

Reply:
Env vars are always strings; the string "Infinity" would work just fine.

However, I think that this PR seems to be doing two things, and it should only be doing one - can you split the "env var" and the "json stringify" parts into separate PRs?

Reply:
Yeap, makes sense. Will do. Thank you